multicluster/cmd/link.go

```diff
@@ -457,11 +465,26 @@ func buildServiceMirrorValues(opts *linkOptions) (*multicluster.Values, error) {
 		return nil, fmt.Errorf("--log-format must be one of: plain, json")
 	}
 
+	if opts.selector != "" && opts.selector != fmt.Sprintf("%s=%s", k8s.DefaultExportedServiceSelector, "true") {
+		if !opts.enableGateway {
+			return nil, fmt.Errorf("--selector and --gateway=false are mutually exclusive, received selector: %s", opts.selector)
+		}
+	}
+
+	if opts.gatewayAddresses != "" && !opts.enableGateway {
+		return nil, fmt.Errorf("--gateway-addresses and --gateway=false are mutually exclusive, received selector: %s", opts.selector)
```

Comment:
Bad copy paste leaving the selector in the error message?

Reply:
Thanks, my bad. On second thought, I left it at "are mutually exclusive", it should be easy to spot any overrides passed in without having to log them.